src/main/java/by/exadel/internship/service/InternshipService.java

```diff
@@ -3,13 +3,16 @@
 import by.exadel.internship.dto.internship.GuestFullInternshipDTO;
 import by.exadel.internship.dto.internship.GuestShortInternshipDTO;
 import by.exadel.internship.dto.internship.UserInternshipDTO;
+import by.exadel.internship.entity.Internship;
 
 import java.util.List;
 import java.util.UUID;
 
 public interface InternshipService {
 
-    GuestFullInternshipDTO getById(UUID id);
+    GuestFullInternshipDTO getGuestFullInternshipDTOById(UUID id);
```

Comment:
GuestFullInternshipDTO guest representation is limited, isn't it? Why it is named as full?

Reply:
because it is maximum full form for Guest, we have also short version

Reply:
I got, this naming is not informative

Reply:
rename GuestShortInternshipDTO to BaseInternshipDTO, and GuestFullInternshipDTO to GuestInternshipDTO